markets/perps-market/contracts/interfaces/ILiquidationModule.sol

```diff
@@ -61,6 +71,14 @@ interface ILiquidationModule {
      */
     function liquidate(uint128 accountId) external returns (uint256 liquidationReward);
 
+    /**
+     * @notice Liquidates an account's margin when no other positions exist.
+     * @dev if available margin is negative and no positions exist, then account margin can be liquidated by calling this function
+     * @param accountId Id of the account to liquidate.
+     * @return liquidationReward total reward sent to liquidator.
+     */
+    function liquidateMarginOnly(uint128 accountId) external returns (uint256 liquidationReward);
```

Comment:
Is there no need to include a check that the margin has already been liquidated? Specifically concerned about the case, where a liquidation is rate limited, and someone calls this many times, during the rate limit.

Reply:
this type of liquidation is never rate limited as there are no positions.  it only gets triggered when there are no positions and your available margin is lower than the debt you need to pay.